packages/frontend-shared/cypress/support/mock-graphql/longBrowsersList.ts

```diff
@@ -10,6 +10,8 @@ export const longBrowsersList = [
     majorVersion: '73',
     info: 'Info about electron browser',
     isFocusSupported: true,
+    isVersionSupported: true,
```

Comment:
Let's add a browser that has `isVersionSupported: false` and a `warning` message, and then add a test that these elements show up in the tooltip for `OpenBrowserList.cy.tsx`. `cy.percySnapshot()` would be good to capture this state as well.